mowgli_etl/pipeline/wdc/wdc_offers_corpus.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass
+from typing import Optional, List
+from pathlib import Path
+from mowgli_etl.pipeline.wdc.wdc_offers_corpus_entry import WdcOffersCorpusEntry
+
+
+@dataclass(init=False)
+class WdcOffersCorpus:
+	entries: Optional[List[WdcOffersCorpusEntry]]
+
+	def __init__(self, *, wdc_json_file_path: Path):
+		self.entries = []
+		for row in open(wdc_json_file_path):
+			self.entries.append(WdcOffersCorpusEntry.from_json(row))
```

Comment:
This is going to blow out memory for the real corpus. Just save the file path to a private member and then read it on demand in the methods.

Reply:
Some indentation issues here.